src/lib/avatar/avatar.ts

```diff
@@ -54,6 +56,9 @@ export const AVATAR_TAG_NAME: keyof HTMLElementTagNameMap = 'forge-avatar';
 export class AvatarComponent extends LitElement implements IAvatarComponent {
   public static styles = unsafeCSS(styles);
 
+  /** @deprecated Used for compatibility with legacy Forge @customElement decorator. */
+  static [CUSTOM_ELEMENT_NAME_PROPERTY] = AVATAR_TAG_NAME;
```

Comment:
Should this have an access modifier?

Reply:
It doesn't matter much to me. I suppose we might as well just make them all `private` so nobody gets the wrong idea about using them since we are intentionally introducing them as deprecated. I'll update it.

Reply:
Updated. Decided to use `public` because that's actually how we intend to use them even though it's only for internal use.